Project.toml

```
@@ -1,7 +1,7 @@
 name = "MultiDocumenter"
 uuid = "87ed4bf0-c935-4a67-83c3-2a03bee4197c"
 authors = ["Sebastian Pfitzner <pfitzseb@gmail.com> and contributors"]
-version = "0.6.1"
+version = "0.7.0"
```

Or `0.6.2`? Not sure if we should consider the "breaking" windows a breaking change or not.

```suggestion
version = "0.6.2"
```

---

0.7.0 makes sense to me.